pkg/node/kubelet.go

```diff
@@ -180,3 +187,21 @@ func loadConfigFile(name string) (*kubeletconfig.KubeletConfiguration, error) {
 	}
 	return kc, err
 }
+
+func loadOSID() (string, error) {
+	readFile, err := os.Open("/etc/os-release")
+	if err != nil {
+		return "", err
```

Comment:
This will break the environment of customers testing on non-rhel os's for GPU reasons. May be we should print a warning and use "unknown"? 

It'd be easy for them to workaround though, but I wanted to highlight it.

Reply:
Any system relying on `systemd` will support `/etc/os-release`, including Debian-derived ones, but I agree I don't see a reason to fail fatally if the OS info cannot be read.

Reply:
Done